### PR TITLE
[3.3.0] 3.3.0 release

### PR DESCRIPTION
**Pull request by mattle24**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
         libudunits2-dev \
         libcairo-dev \
-        libgdal-dev \        
+        libgdal-dev \
         libgeos-dev \
         libglu1-mesa-dev \
         libx11-dev \
-        mesa-common-dev \        
+        mesa-common-dev \
+				libsodium-dev \
```
Comment (patr1ckm) [inline Dockerfile:14]: de-indent and alphabetize, otherwise great!

Comment (mattle24) [reply Dockerfile]: Fixed it! Dang tabs being inaccurate.

**Comment by patr1ckm** [comment]
LGTM!

**Comment by patr1ckm** [comment]
Actually, one sec, changelog

**Comment by mattle24** [comment]
Btw, I built this locally and it seems fine, but I don't think Dockerhub is set to build `dev-` branches because this tag does not appear on Dockerhub

**Comment by patr1ckm** [comment]
Hm, this must have a been a recent change. Thanks for flagging!

**Pull request merged**